<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgHealthOptions</class>
 <widget class="QDialog" name="dlgHealthOptions">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesundheits-Faktoren</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>196</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextSport1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sport 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextSport2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sport 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextStress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stress</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextIllness">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Krankheit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextFemale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vor Periode</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnSport1Factor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnSport2Factor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnStressFactor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnIllnessFactor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnFemaleFactor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextSober">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃ¼chtern</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnSoberFactor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextOther">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Andere</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnOtherFactor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPercent7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgHealthOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgHealthOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
